src/GameOptions/ui/SystemPage.tsx

```diff
@@ -132,6 +138,20 @@ export const SystemPage = (): React.ReactElement => {
           tooltip={<>The time (in seconds) between each autosave. Set to 0 to disable autosave.</>}
           marks
         />
+        <OptionsSlider
+          label="Tail render intervall (ms)"
+          initialValue={tailrenderInterval}
+          callback={handleTailIntervalChange}
+          step={200}
+          min={200}
```

Comment:
I think we should have min go down to 100, possibly even 50. People might come up with interesting animations or something at high rerenders, and I don't think it should hurt performance too badly (we'll test). But the important part is that it's a setting, so it's users choice if they want to be extreme.

Reply:
then with a smaller step or is there a way to change the step size depending on the current value? (like 50-200 steps of 10, 200-2000 steps of 200, 2000-5000 steps of 1000 or so)

Reply:
I'm pretty sure step size has to be fixed (limitation of the actual DOM slider). Options would be to decrease the step size, or it might work out if you leave step at 200 since initial is 1000, giving steps at 1000, 800, 600, 400, 200, 50?

Reply:
i tried min 50 with 200 steps 
sadly ends with 50-250-450  etc after the first move away from 1000

Reply:
Dang. Well, go with whatever you think works best then. I'd lightly prefer the ability to have a smaller min refresh, but I'm not dead set on it.

Reply:
i hoped "we" could get a ns method at some point to set it to whatever the player wants to use
(and to reapply a changed setting on already existing tails?)
but thats above my react level

Reply:
IMO we'd want that to apply per-tail and not affect the global setting, but I'm okay with that as a justification for the global only going to 200.

Reply:
oh yes totaly meant as per tail like the other tail functions